### PR TITLE
feat: add aws scraper

### PR DESCRIPTION
**Pull request by yashmehrotra**
Closes: https://github.com/flanksource/mission-control-registry/issues/24

### CODE REVIEW

File: charts/aws/values.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      configMapKeyRef:
+        key: ''
+        name: ''
+      secretKeyRef:
+        key: ''
+        name: ''
+  region: ''
+  endpoint: ''
+  skipTLSVerify: ''
+  assumeRole: ''
```
Comment (yashmehrotra) [inline charts/aws/values.yaml:41]: @moshloop Should we take all the connection details as input and let the user pass what they want ?

File: charts/aws/values.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  table: athena_database_table_name
+
+inventory: true
+
+patchDetails: true
+
+patchStates: true
+
+trustedAdvisorCheck: false
+
```
Comment (yashmehrotra) [inline charts/aws/values.yaml:64]: Also, keeping all the keys in root, is it okay that way or should everything be prefixed with `scraper.` ?

File: charts/aws/values.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+# AWS regions to pull from
+regions:
```
Comment (moshloop) [inline charts/aws/values.yaml:66]: I think we should comment these out, and make region mandatory

File: charts/aws/values.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+compliance: true
+
+# Scrape cost reports and match them with catalog items
+costReporting:
+  enabled: false
+  database: athena_database_name
+  region: athena_database_region
+  s3BucketPath: s3_bucket_path_for_athena_query_results
+  table: athena_database_table_name
```
Comment (moshloop) [inline charts/aws/values.yaml:55]: I don't think we should  have these dummy values

File: charts/aws/values.yaml [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+- eu-west-3
+- eu-south-2
+- eu-north-1
+- eu-central-2
+- il-central-1
+- me-south-1
+- me-central-1
+- sa-east-1
+- us-gov-east-1
+- us-gov-west-1
```
Comment (moshloop) [inline charts/aws/values.yaml:97]:
```suggestion
regions: []
```

**Pull request merged**